src/event_queue.hpp

```diff
@@ -32,9 +32,11 @@ struct postsynaptic_spike_event {
         return l.target==r.target && l.time==r.time && l.weight==r.weight;
     }
 
+    /*
     friend bool operator<(postsynaptic_spike_event l, postsynaptic_spike_event r) {
         return l.time<r.time;
     }
+    */
```

Comment:
Of course. I must have commented it out as a hack to check if it was used, then forgot to actually remove the code.